src/common/StatementMetadata.cpp

```diff
@@ -167,7 +167,7 @@ const char* StatementMetadata::getPlan(bool detailed)
 
 	if (plan->isEmpty())
 	{
-		UCHAR info[] = {detailed ? isc_info_sql_explain_plan : isc_info_sql_get_plan};
```

Comment:
Oh, sorry, I don't know c++ so much.
explicit types casts means modify

from

    (UCHAR)ics_info_sql_explain_plan

to

    UCHAR(isc_info_sql_explain_plan)

like this ?